services/keyboard.md

```diff
@@ -0,0 +1,39 @@
+# Keyboard
+
+    identifier: 0x18b05b6a
+    
+A keyboard that can send keystrokes.
+
+The codes for the key (selectors) is defined in the [HID Keyboard
+specification](https://usb.org/sites/default/files/hut1_21.pdf), chapter 10 Keyboard/Keypad Page, page 81.
+    
+## Commands
+
+    flags Modifiers : u8 {
```

Comment:
you would need to use different numbers (separate bits) to make this flags; given that there's already 8 of them, I suggest u16

Reply:
do they even treat modifiers separately in the HID spec? maybe we also don't have to

Reply:
oh and the key is GUI not GUID

Reply:
Ooops, copy paste issue. The HID spec reserves a single range for the modifiers so u8 is enough